generateSchema.js

```diff
@@ -437,14 +439,25 @@ function processModels() {
         return model && model.typeId;
     }
     function isSchemable(modelName) {
-        return modelName.match(/Catalog(Item|Group|Member)\.js$/) &&                   
+        return modelName.match(/Catalog(Item|Group|Member)\.js$/) &&
               !modelName.match(/(ArcGisMapServerCatalogGroup|addUserCatalogMember)/);
```

Comment:
It was deprecated at one point, kind of like `AsyncFunctionResultCatalogItem` is now.  But Arthur resurrected it recently, so it should no longer be excluded.